packages/mobile/components/Confirmation.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
```

Comment:
Not sure I see this as a component, wouldnt you move it to a generic view like unlock stronghold, and all it something like `ConfirmView` or so?

Reply:
But unlock stronghold is a component as well, at least at the moment. Do you mean we should move both into the view's folder? Maybe views/generics or views/dashboard/generics?